test/math_test.py

```diff
@@ -7,3 +7,15 @@ def test_double_list():
     assert ms.double_list([2]) == [4]
     assert ms.double_list([1, 2]) == [2, 4]
     assert ms.double_list([1, 2, 3]) == [2, 4, 6]
+
+def test_multiply_list():
+    assert ms.multiply_list([], 1) == []
+    assert ms.multiply_list([1], 0) == [0]
+    assert ms.multiply_list([1], 1) == [1]
+    assert ms.multiply_list([1], 2) == [2]
+    assert ms.multiply_list([1, 2], 0) == [0, 0]
+    assert ms.multiply_list([1, 2], 1) == [1, 2]
+    assert ms.multiply_list([1, 2], 2) == [2, 4]
+    assert ms.multiply_list([1, 2, 3], 0) == [0, 0, 0]
+    assert ms.multiply_list([1, 2, 3], 1) == [1, 2, 3]
+    assert ms.multiply_list([1, 2, 3], 2) == [2, 4, 6]
```

Comment:
A falta de um linha em branco no final deste arquivo está gerando o erro `test/math_test.py:21:55: W292 no newline at end of file`.